### ISSUE TITLE
NVDA unable to use 'escape' to exit the autocomplete

### ISSUE THREAD

**Issue by nickcolley** [alphagov/accessible-autocomplete#346]
This issue is from a May 2019 external accessibility audit report.

**Issue ID**: DAC_4

> "I found that I was unable to use the ‘Escape’ key to exit the autocomplete and was only able to exit using the ‘Tab’ key or by making a selection from one of the available options. This could mean that some users become trapped within the autocomplete and are unable to exit it. Please note that this issue did not occur using JAWS and VoiceOver for iPhone."

**Comment by nickcolley** [comment]
I've not had the time to reproduce this yet, so this requires investigation to see if this is something we can resolve.
